cmd/documentation/documentation.go

```diff
@@ -469,21 +348,32 @@ func GetTemplateFiles() (*template.Template, error) {
 	return tmpl, filepath.Walk(toolDir, walkFn)
 }
 
-// GetContributorList fetches a list of contributors from the github api
-// endpoint
-func GetContributorList(repo string, verbose bool) ([]Contributor, error) {
-	contents, err := common.SendHTTPRequest(context.TODO(),
-		http.MethodGet,
-		repo+GithubAPIEndpoint,
-		nil,
-		nil,
-		verbose)
-	if err != nil {
-		return nil, err
-	}
+// GetContributorList fetches a list of contributors from the Github API endpoint
+func GetContributorList(ctx context.Context, repo string, verbose bool) ([]Contributor, error) {
+	var contributors []Contributor
+	vals := url.Values{}
+	vals.Set("per_page", defaultGithubAPIPerPageLimit)
+	page := 1
+	for {
+		vals.Set("page", strconv.Itoa(page))
 
-	var resp []Contributor
-	return resp, json.Unmarshal(contents, &resp)
+		contents, err := common.SendHTTPRequest(ctx, http.MethodGet, common.EncodeURLValues(repo+GithubAPIEndpoint, vals), nil, nil, verbose)
+		if err != nil {
+			return nil, err
+		}
+
+		var resp []Contributor
+		if err := json.Unmarshal(contents, &resp); err != nil {
+			return nil, err
+		}
```

Comment:
🤷 
If len(contributors) divides equally into your pages, then will we get an error after the last page?
Is there anything other than number to tell us we were done?

Reply:
Nope, we'll get an empty result on the next page which means the append will be a no-op and break still. There's a header returned which has the max page limit but not high enough ROI to implement since this works